policy/predicate/file.go

```diff
@@ -140,6 +192,14 @@ func (exp ComparisonExpr) MarshalText() ([]byte, error) {
 	return []byte(fmt.Sprintf("%s %d", op, exp.Value)), nil
 }
 
+func (exp ComparisonExpr) MarshalTextToString() string {
+	res, err := exp.MarshalText()
+	if err != nil {
+		return "Unknown"
```

Comment:
```suggestion
		return fmt.Sprintf("?? (op:%d) %d"), exp.Op, exp.Value) 
```

This will give at least a bit of information about what's wrong.